.circleci/config.yml

```diff
@@ -97,7 +97,11 @@ aliases:
       fullstack) make test CHECKSTYLE=0 FULLSTACK=1           PROVE_ARGS="$HARNESS -v t/full-stack.t" RETRY=3;;
       scheduler) make test CHECKSTYLE=0 SCHEDULER_FULLSTACK=1 PROVE_ARGS="$HARNESS -v t/05-scheduler-full.t";;
       developer) make test CHECKSTYLE=0 DEVELOPER_FULLSTACK=1 PROVE_ARGS="$HARNESS -v t/33-developer_mode.t";;
-      build.docs) COMMIT_AUTHOR_EMAIL=skynet@open.qa GEM_HOME=$(pwd)/.gem script/generate-documentation $encrypted_e2c381aa6b8c_key $encrypted_e2c381aa6b8c_iv;;
+      build.docs) sudo zypper -n install hub
+                  echo 'echo $GITHUB_TOKEN' > $(pwd)/.git-askpass
+                  export GIT_ASKPASS=$(pwd)/.git-askpass
+                  chmod +x $(pwd)/.git-askpass
+                  GEM_HOME=$(pwd)/.gem PUBLISH=1 PULL_REQUEST_TARGET="${CIRCLE_PROJECT_USERNAME}:gh-pages" bash -x script/generate-documentation https://token@github.com/openqabot/openQA.git gh-pages-daily
```

Comment:
Maybe this should just be its own job? It doesn't seem to share much with the test runs now